.travis.yml

```diff
@@ -84,12 +88,3 @@ script:
   - if [[ "$PHP_COVERAGE" = "TRUE" ]]; then wget https://scrutinizer-ci.com/ocular.phar; fi
   - if [[ "$PHP_COVERAGE" = "TRUE" ]]; then php ocular.phar code-coverage:upload --format=php-clover clover.xml; fi
 
-matrix:
```

Comment:
why did you remove the matrix? We have at least three builds: the latest release of the Nextcloud server, its current master branch and the js tests.